cypress/integration/lists_test/enrichment_test.js

```diff
@@ -0,0 +1,99 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
+    if (err.message.includes('Can not create edge')) {
+        // Cytoscape.js tools are throwing this - ignore for now.
+        return false
+    }
+})
+
+describe("Enrichment Test", function(){
+    beforeEach(function(){
+        cy.visit("/");
+        cy.createListFromFile("all_genes.csv","All genes"); //This file is downloaded from querybuilder.
+        cy.createListFromFile("enrichment_gene.csv","Enrichment analysis genes");
+        cy.get(".enrichment").should("exist").within(() => {
+            cy.get(".dropdown").click();
+            cy.get(".list-selection").eq(0).click();
```

Comment:
It appears you are setting the background population for every test. Can you instead only do this for one test, which is to specifically test the background population? When it isn't set, the enrichment will use all the genes in the mine as background population.